src/RestaurantButton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export const RestaurantButton = props => {
+  return (
+    <>
```

Comment:
But here we do need to use the fragments, nice! Good idea to use this than another DOM element 👍